c_glib/arrow-glib/basic-array.cpp

```diff
@@ -643,6 +643,50 @@ garrow_array_dictionary_encode(GArrowArray *array,
   return garrow_array_new_raw(&arrow_dictionary_encoded_array);
 }
 
+/**
+ * garrow_array_count:
+ * @array: A #GArrowArray.
+ * @options: (nullable): A #GArrowCountOptions.
+ * @error: (nullable): Return location for a #GError or %NULL.
+ *
+ * Returns: The number of target values on success, %NULL on error.
```

Comment:
Should it be `0 on error.`?

Reply:
Good catch!
We should not use "0 on error" because 0 doesn't always indicate error.
I update the document to users should check `error`.

Reply:
I see. Thanks!